projects/convert pdf to text/converter1.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep  9 16:38:39 2020
+
+@author: Admin
+"""
+
+import PyPDF2
+
+pdfobj = open('/path/to/pdf', 'rb') #Provide the path for your pdf here
```

Comment:
ask user input for pdf file
```suggestion
file_path =input('FilePath:')
pdfobj = open(file_path 'rb') #Provide the path for your pdf here
```